modules/solid_mechanics/include/postprocessors/InteractionIntegral.h

```diff
@@ -54,6 +58,12 @@ class InteractionIntegral : public ElementIntegralPostprocessor
   bool _has_symmetry_plane;
   bool _t_stress;
   Real _poissons_ratio;
+  unsigned int _ring_index;
+  unsigned int _ring_first;
+  MooseEnum _q_function_type;
+  std::vector<Real> _q_elem;
```

Comment:
How about calling these `_q_curr_elem`, `_phi_curr_elem`, etc, just to be a little more clear.